src/content/docs/apm/apm-ui-pages/monitoring/view-slow-query-details.mdx

```diff
@@ -10,19 +10,17 @@ metaDescription: 'APM: How to view slow query data, and how to enable and config
 redirects:
   - /docs/apm/applications-menu/monitoring/viewing-slow-query-details
   - /docs/apm/applications-menu/monitoring/view-slow-query-details
-freshnessValidatedDate: never
+freshnessValidatedDate: 2024-06-07 
 ---
 
-In APM, when [transaction traces](/docs/apm/transactions/transaction-traces/introduction-transaction-traces) are collected, there may be additional <DoNotTranslate>**Slow query**</DoNotTranslate> data available on the [<DoNotTranslate>**Databases**</DoNotTranslate>](/docs/apm/applications-menu/monitoring/databases-slow-queries-page) page. Let's look at what slow query data is, where to find it, and how to configure it.
 
-You can also use histograms and heatmaps to visualize the distribution of values for different metrics. This is useful to understand slow page performance. To learn how, watch this short video (4:18 minutes).
+import apmApmDatabasesUi from 'images/apm_screenshot-full_apm-databases-ui.webp'
+
 
-<Video
-  id="Gh4Km9aJoBE"
-  type="youtube"
-/>
 
-## Slow query data samples [#about]
+In APM, when [transaction traces](/docs/apm/transactions/transaction-traces/introduction-transaction-traces) are collected, there may be additional <DoNotTranslate>**Slow query**</DoNotTranslate> data available on the [<DoNotTranslate>**Databases**</DoNotTranslate>](/docs/apm/applications-menu/monitoring/databases-slow-queries-page) page. Let's look at what slow query data is, where to find it, and how to configure it.
```

Comment:
I'd avoid the passive here. Maybe this? `In APM, when we collect [transaction traces](/docs/apm/transactions/transaction-traces/introduction-transaction-traces), we might also find some extra <DoNotTranslate>**Slow query**</DoNotTranslate> data on the [<DoNotTranslate>**Databases**</DoNotTranslate>](/docs/apm/applications-menu/monitoring/databases-slow-queries-page) page.`

I also add this page: [View slow query details](https://docs.newrelic.com/docs/apm/apm-ui-pages/monitoring/view-slow-query-details/)

Reply:
@nbaenam thanks for the feedback! I took your advice about the active voice. I also clarified that it's the page that displays the data:

In APM, when we collect [transaction traces](/docs/apm/transactions/transaction-traces/introduction-transaction-traces), we might also find some extra <DoNotTranslate>**Slow query**</DoNotTranslate> data and display it on the [<DoNotTranslate>**Databases**</DoNotTranslate>](/docs/apm/apm-ui-pages/monitoring/databases-page-view-operations-throughput-response-time/) page. Let's look at what slow query data is, where to find it, and how to configure it.